### PR TITLE
gh-2717: Fixed broken image on about page.

### LINKED ISSUE #2717: Broken github "fork me" image on about page
**Describe the bug**

**To Reproduce**
1. Go to [About Page](https://online-go.com/docs/about)
2. Top right

**Expected behavior**
Expected to see "Fork on Github" banner

**Screenshots**
example

**Desktop (please complete the following information):**
 - OS: macOS, iOS, Windows
 - Browser Safari, Chrome 

**Additional context**
Broken image. This has likely been broken for some time now. I recommend removing it altogether (links to GitHub exist in the navbar and developer page), adding a footer (to all views except for game view) and adding it there, or rewriting the about page to incorporate GitHub and social media references/icons to the about page body, beneath the paragraph.

I'll open a PR to remove the broken image, for now, and open a separate feature issue depending on feedback

### PR DESCRIPTION
**Pull request by pygeek**
Fixes #2717 

## Proposed Changes

Remove broken image. This has likely been broken for some time now. I have feature recommendations (ie: adding a footer to all views, except for game view, which incorporates GitHub, discourse, and possibly other social media icons/links) in the referenced issue.

### CODE REVIEW

**Review by anoek — CHANGES_REQUESTED** [review]
This seems to only remove the link not fix the issue, could we fix the broken image so we have our banner still?

**Comment by pygeek** [comment]
> This seems to only remove the link not fix the issue, could we fix the broken image so we have our banner still?

Sure. As a side note, the banner has been broken for 5 years.

**Comment by pygeek** [comment]
> This seems to only remove the link not fix the issue, could we fix the broken image so we have our banner still?

Sure. Do you want me to use the original image as found in the archive (resolution is low), or implement a pure css solution?

As a side note, the banner has been broken for 5 years.

**Comment by anoek** [comment]
> Sure. Do you want me to use the original image as found in the archive (resolution is low), or implement a pure css solution?

Awesome thanks for taking that on :) CSS sounds cleanest and most future proof, but whichever

**Comment by pygeek** [comment]
> Awesome thanks for taking that on :) CSS sounds cleanest and most future proof, but whichever

I committed the original image for now, but will update this PR to use CSS.

**Comment by pygeek** [comment]
Would it be possible to push this to the CDN to ensure my changes work as expected in production?

**Review by anoek — APPROVED** [review]

**Comment by anoek** [comment]
Sure thing, https://cdn.online-go.com/5.1/img/fork_me_right_gray_6d6d6d.svg

File: src/views/docs/About.tsx [hunk 1]
```diff
@@ -35,15 +35,15 @@ export class About extends React.Component<{}, any> {
...
         const server_version = data.get("config")?.version;
+        const cdn_release = data.get("config.cdn_release");
 
         return (
             <div className="About container">
                 <a href="https://github.com/online-go/online-go.com/">
                     <img
                         className="github-link"
-                        src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67"
+                        src={`${cdn_release}/img/fork_me_right_gray_6d6d6.svg`}
```
Comment (anoek) [inline src/views/docs/About.tsx:45]: Missing a `d` at the end of that color code, after that I think everything will work for you

**Comment by pygeek** [comment]
> Sure thing, https://cdn.online-go.com/5.1/img/fork_me_right_gray_6d6d6d.svg

Thanks for uploading the image, I was able to test my changes and fix the typo. Can CI/CD be updated to deploy, and reference artifacts in the build, depending on the env context (ie: local, staging, prod)?

**Comment by anoek** [comment]
> CI/CD be updated to deploy, and reference artifacts in the build, depending on the env context (ie: local, staging, prod)?

Probably not since it's a public repo, I'd like to have a human in the middle there sanity checking that.

**Comment by anoek** [comment]
Thanks for the fix!

**Pull request merged**